ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -227,15 +228,17 @@ void fetchAndRunCommands() {
     }
   }
 
+  @SuppressFBWarnings("DMI_INVOKING_TOSTRING_ON_ARRAY")
```

Comment:
Why is this suppression here? `CommandId` has a `toString()` method and doesn't appear to be an array.

Reply:
I believe the JsonValue is causing the Array Warning here: https://github.com/confluentinc/ksql/blob/954cff98077f98746be40ec6650296b2328e34d0/ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/CommandId.java#L88

This is replaced by getAndDeserializeCommandId() from 6.1.x